.husky/pre-commit

```diff
@@ -1,8 +1,15 @@
 #!/bin/sh
+echo "Running pre-commit hook"
+
 . "$(dirname "$0")/_/husky.sh"
 
 set -e
 
```

Comment:
why do we have `fnm use` twice?

Reply:
wooops, that's a rebase issue, fixing now, thanks for the spot @bryophyta